CHANGELOG.rst

```diff
@@ -104,6 +104,8 @@ Nieuwe features
   uit de API correct worden overgenomen in de lokale versie van de catalogus.
 * [:taiga-us:`3578`, :pr:`2043`]: Nieuw Front-end ontwerp voor CMS plug-in 'Balie Afspraken' gebouwd met
   design-tokens volgens het NLDS principe.
+* [:taiga-us:`3609`, :pr:`2062`]: Nieuw accordion web component toegevoegd dat gebruikt gaat worden voor
```

Comment:
Changelog moet gaan over wat is opgeleverd, niet wat nog gaat gebeuren. 
Dus mag worden "Nieuw accordion web component toegevoegd dat gebruikt wordt in Mijn Afval"